2023 09 05

KawakamiShunki opened the pull request:
ご指摘頂いていた不要な改行部分を修正いたしました。
確認のほどよろしくお願いいたします。

KawakamiShunki committed:
不要な改行修正

KawakamiShunki committed:
Merge branch 'main' of github.com:KawakamiShunki/nintendo into 2023-09-05

# Conflicts:
#	untitled/nintendo.html

KawakamiShunki committed:
page_main以下作成

KawakamiShunki committed:
page_main以下作成

KawakamiShunki committed:
09.05

KawakamiShunki commented:
page_main以下をHTML、CSSまで作成。
product_cardLists以下のHTMLを途中まで作成しました。
確認のほど、よろしくお願いします。

sugiyama-kotarou commented on untitled/nintendo.html:
```diff
@@ -1,645 +1,1032 @@
+                <p class="header_suggestLabel">よく検索されているワード</p>
+                <ul class="header_suggestItems">
+                  <li class="header_suggestItem">
+                    <a href="https://store-jp.nintendo.com/search/?=amiibo スプラトゥーン">amiibo
```
この部分に関しては、下記のような形に修正しましょう
`<a href="https://store-jp.nintendo.com/search/?=amiibo スプラトゥーン">`
`  amiiboスプラトゥーン`
`</a>`

sugiyama-kotarou reviewed (commented):
images/2.svgに関してですが、ナンバリングするだけの命名はやめましょう。
ceroの画像であれば、cero.svgなどの命名にしてください。
https://qiita.com/kazukichi/items/ecf8e1caa040b704ceab

sugiyama-kotarou commented on untitled/resource/css/nintendo.css:
```diff
@@ -48,312 +48,312 @@ body {
 }
```
中身にstyleを書いていない、使用していないclassは削除しましょう
レスポンシブに使用している場合でもpc側のclassは不要です。

sugiyama-kotarou commented on untitled/resource/css/nintendo.css:
```diff
@@ -48,312 +48,312 @@ body {
+    overflow: visible !important;
+    height: auto !important;
+  }
 }
```
プロジェクトでは!importantは出来るだけ使用しないルールなので、!importantを使用しない書き方に修正してください。
https://csshtml.work/important-not-recommended/

sugiyama-kotarou commented on untitled/resource/css/nintendo.css:
```diff
@@ -48,312 +48,312 @@ body {
-    padding-top: 17px;
+  position: relative;
+  padding-top: 17px;
 }
```
ここclass名が無いので、class名を定義してください。
不要なら削除してください。

sugiyama-kotarou commented on untitled/resource/css/nintendo.css:
```diff
@@ -364,768 +364,1491 @@ body {
 
 }
 
```
不要なら削除してください。
中身の無いclassその他全て確認してください。

sugiyama-kotarou commented on untitled/resource/css/nintendo.css:
```diff
@@ -1,39 +1,39 @@
+  display: block;
 }
 
 @media (min-width: 768px) {
```
@media (min-width: 768px) {
}

レスポンシブの場合、上記で囲むと思いますが、class毎に囲んで書かなくても、下記の様に書くことが出来ます。
なので、都度@media (min-width: 768px) {}は書かないようにしてください。
PCのスタイルを1000行書いたら、その下に@media (min-width: 768px) {}を記述して、その中にスマホサイズのスタイルを○○行書くような形にしてください。
上がpc用のスタイル、下がスマホ用のスタイルというような書き方になります。
cssファイルを見たときに、下がスマホ用だなというように分かりやすくメンテナンス性が上がります。
![image](https://github.com/KawakamiShunki/nintendo/assets/81726492/0c23f8dc-dbb6-452b-bade-ecc65882c03a)

sugiyama-kotarou commented on untitled/resource/css/nintendo.css:
```diff
@@ -1,39 +1,39 @@
 }
 
 @media (min-width: 768px) {
-    .banner_inner {
```
ちなみにレスポンシブ画面は、後で良いという話になっているので、とりあえずnintendoページのstyleを丸々コピーペーストするのはやめましょう
とりあえずレスポンシブのスタイルは全て削除してください。

sugiyama-kotarou commented on untitled/resource/css/nintendo.css:
```diff
@@ -364,768 +364,1491 @@ body {
+  padding-inline: 6px;
+  border-width: 2px;
+  border-style: outset;
+  border-color: buttonborder;
```
エラー出ているようなのですが、正しいスタイル書けてますか？
必要無ければ削除してください。
phpstormの右上にエラーマークがあり、そこからエラーが確認出来ます。
![image](https://github.com/KawakamiShunki/nintendo/assets/81726492/6ce50a43-6b49-491e-9b9b-e4436ef1250f)

sugiyama-kotarou commented on untitled/resource/css/nintendo.css:
```diff
@@ -364,768 +364,1491 @@ body {
+  border-color: inherit;
+}
 
 
```
基本的にファイルは最後の一行を改行する必要がルールで決められています。
`// ファイルの末尾は改行し空行をいれる`
下記伝え忘れていたので、意識してコーディングしてください。
Laravelのコード書くときにもこちらを確認してください。
https://qiita.com/speedstar18fct/items/c5966cb761596fc790bc

sugiyama-kotarou commented:
プルリクエストのレビューを出すときのwikiがありましたので、共有しておきます。
こちら一度目を通しておいてください。
https://valeur.backlog.jp/alias/wiki/857178

KawakamiShunki committed:
・SVGの命名を変更
・改行がおかしかった場所を修正
・使用してなかったstyleとclassを削除
・HTMLに直書きしていたstyleに関してもCSSへ記載を移動
・importantを削除
・class名の記載のなかったstyleにクラス名を記載
・@mediaで指定した全てのレスポンシブ向けstyleを削除
・ファイルの末尾に空行を追加

KawakamiShunki commented:
上記コメントの修正点及び、PR前の確認事項に抵触しそうな箇所を修正しました。
確認のほどよろしくお願いいたします。

・SVGの命名を変更
・改行がおかしかった場所を修正
・使用してなかったstyleとclassを削除
・HTMLに直書きしていたstyleに関してもCSSへ記載を移動
・importantを削除
・class名の記載のなかったstyleにクラス名を記載
・@mediaで指定した全てのレスポンシブ向けstyleを削除
・ファイルの末尾に空行を追加

sugiyama-kotarou commented on untitled/resource/css/nintendo.css:
```diff
@@ -18,6 +18,11 @@ body {
 
+.page_banner > a {
+  background-color: #f0f0f0;
+"
```
不要なダブルクォートが入っているので削除お願いします。

KawakamiShunki committed:
不要なダブルクォートを削除
